src/components/webexWalkin/WebexWalkin.ts

```diff
@@ -103,10 +102,11 @@ export namespace WebexWalkin {
       return this.progressValue
         ? html`
             <md-progress-bar
-              .type=${"determinate"}
+              class="walkin-progress-bar"
+              type="determinate"
               .value=${this.progressValue}
-              .displayFormat=${"none"}
-              .dynamic=${true}
+              displayFormat="none"
```

Comment:
do we not have a non-camel case attribute name for the progress bar? seems like an S6 bug for later

Reply:
I'll create a Jira bug for that. currently doesn't exist.